CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # 4.12.3 (2022-01-03)
 
 - Added support for wrapped text in custom components. [#177](https://github.com/blackbaud/skyux-ag-grid/pull/177)
+- Fixed an error with the validator popover change detection. [#180](https://github.com/blackbaud/skyux-ag-grid/pull/180)
```

Comment:
Same question as in https://github.com/blackbaud/skyux-ag-grid/pull/179 about whether we really need `popover` in there. And like that PR, I'll go ahead and approve assuming we do need it, but just let me know if you remove it so that I can circle back and re-approve. Thanks!